apps/src/templates/teacherDashboard/LoginTypePicker.jsx

```diff
@@ -79,18 +99,18 @@ class LoginTypePicker extends Component {
             <WordLoginCard onClick={setLoginType} />
             <EmailLoginCard onClick={setLoginType} />
           </CardContainer>
+          {!hasThirdParty && (
+            <div>
+              {i18n.thirdPartyProviderUpsell() + ' '}
+              <a href="https://support.code.org/hc/en-us/articles/115001319312-Setting-up-sections-with-Google-Classroom-or-Clever">
+                {i18n.learnHow()}
+              </a>
+              {' ' + i18n.connectAccountThirdPartyProviders()}
+            </div>
+          )}
         </div>
-        {!hasThirdParty && (
-          <div>
-            {i18n.thirdPartyProviderUpsell() + ' '}
-            <a href="https://support.code.org/hc/en-us/articles/115001319312-Setting-up-sections-with-Google-Classroom-or-Clever">
-              {i18n.learnHow()}
-            </a>
-            {' ' + i18n.connectAccountThirdPartyProviders()}
-          </div>
-        )}
-        <DialogFooter>
```

Comment:
We don't need the DialogFooter anymore?

Reply:
@annaxuphoto Thank you! I'll make sure to attach recordings when that fits better! The DialogFooter has specific absolute positioning styling that was causing issues with the scrolling bug, and I didn't want to modify the DialogFooter itself to mess up other implementations of DialogFooter.